### PR TITLE
Release process for python packages

### PR DESCRIPTION
**Pull request by fsoubelet**

### CODE REVIEW

File: docs/packages/development/howto_release.md [hunk 1]
```diff
@@ -1,7 +1,17 @@
-# How to make a release of a python package
+# Release Process of OMC Packages
 
-!!! todo
-    Detailed description. Remove GIF.
+Our packages' release process to PyPI is automated through [Github Actions][github_actions]{target=_blank}, and triggered by manually creating a release from the Github repository.
 
+When the final changes for a release, including updating the package's version number, have been merged into the `master` branch, the release process is as follows:
```
Comment (mihofer) [inline docs/packages/development/howto_release.md:6]: update changelog too?

Comment (fsoubelet) [reply docs/packages/development/howto_release.md:6]: I though that was included in "changes for a release".

Comment (mihofer) [reply docs/packages/development/howto_release.md:6]: i'd think explicitly mentioning it might be good, assuming that it's an easy thing to forget ..

**Review by mihofer — APPROVED** [review]
good to go from my side

**Pull request merged**